packages/react-components/react-tabster/etc/react-tabster.api.md

```diff
@@ -68,8 +68,8 @@ export const useFocusFinders: () => {
     findAllFocusable: (container: HTMLElement, acceptCondition?: ((el: HTMLElement) => boolean) | undefined) => HTMLElement[];
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26372&commitId=07f9f6673ea1a659cc09bb64b92928c9a0ea4d33&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1774 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Avatar Converged.size+image+badge.normal.chromium.png|1| Changed|

</details>


<div id="vrtCommentfluentuiv9"/>